src/content/docs/infrastructure/host-integrations/host-integrations-list/ansible-controller-integration.mdx

```diff
@@ -0,0 +1,190 @@
+
+<img
+  title="Ansible Controller"
+  alt="A screenshot depicting the Ansible Controller dashboard"
+  src={infrastructureAnsibleControllerDashboard}
+/>
+
+<figcaption>
+After setting up the integration with New Relic, see your data in dashboards like these, right out of the box.
+</figcaption>
+
+## Install the infrastructure agent [#infra]
+
+The New Relic infrastructure agent is the foundation for getting your Ansible Automation Platform data into New Relic. If you haven't already done so, install the agent using one of these options:
+
+
+* Our [guided install](https://one.newrelic.com/nr1-core?state=4f81feab-35f7-e97e-9903-52510f8542bd) is a CLI tool that inspects your system and installs the infrastructure agent alongside the application monitoring agent that best works for your system. To learn more about how our guided install works, check out our [Guided install overview](/docs/infrastructure/host-integrations/installation/new-relic-guided-install-overview).
+* If you'd rather install our infrastructure agent manually, you can follow a tutorial for manual installation for [Linux](/docs/infrastructure/install-infrastructure-agent/linux-installation/install-infrastructure-monitoring-agent-linux), [Windows](/docs/infrastructure/install-infrastructure-agent/windows-installation/install-infrastructure-monitoring-agent-windows/), or [macOS](/docs/infrastructure/install-infrastructure-agent/macos-installation/install-infrastructure-monitoring-agent-macos/).
+
+
+## Integrate Red Hat Ansible Automation Platform with New Relic [#integrate]
+
+Once installed, the infrastructure agent can ingest data from your app and send it to New Relic, but you still need to integrate – or establish a line of communication – between the agent and your app. Once integrated, you can get started with an out of the box monitoring solution for your Ansible Automation Platform app. 
+
+<CollapserGroup>
+<Collapser
+    id="step1"
+    title="Step 1. Create a bearer token on Ansible Automation Platform"
+    >
+Once logged into Ansible Automation Platform, navigate to the **Users** section, select **admin** and then add a **token** to the user. Copy the token and save it in your desired path.
+
+Here is an example token: 
+
+Create a file named "bearer_token_file" in your desired path.
+
+```
+touch /home/ansible-automation-patform/bearer_token_file
+
+```
+And paste your token in the created file.
+
+```
+nano /home/ansible-automation-platform/bearer_token_file
+<paste your token>
+```
+
+Save and exit the nano editor
+
+</Collapser>
+
+<Collapser
+    id="step2"
+    title="Step 2. Configuring Prometheus OpenMetrics integration"
+    >
+
+Update the fields listed below:
+    a. `cluster_name: “YOUR_DESIRED_CLUSTER_NAME”`
+    b. `bearer_token_file: “BEARER_TOKEN_FILE_PATH`
+    c.  `urls: [“https://YOUR_HOST_IP:443/api/v2/metrics/?format=txt”]`
+    d. `Insecure_skip_verify: true`
+    4. Your `nri-prometheus-config.yml` file should look like this:
```

Comment:
Kindly update the lines 70 to 75, with the below text

1. Create a file named “nri-prometheus-config.yml” in the below path
Path: /etc/newrelic-infra/integrations.d/nri-prometheus-config.yml

2. Use our [configuration template](https://github.com/newrelic/nri-prometheus/blob/main/configs/nri-prometheus-config.yml.sample) to update the created file named "nri-prometheus-config.yml"

3. Update the below fields
    a. `cluster_name: “<YOUR_DESIRED_CLUSTER_NAME>”`
    b. `bearer_token_file: “<BEARER_TOKEN_FILE_PATH>”`
    c. `urls: [“https://<YOUR_HOST_IP>:443/api/v2/metrics/?format=txt”]`
    d. `Insecure_skip_verify: true`

4. Your `nri-prometheus-config.yml` file should look like below

Reply:
Done!